src/config/config.js

```diff
@@ -1 +1,2 @@
-module.exports.SEED = "JSI5A6P8D9J2SAPIDJSAIPDJSAIPDJASPI48413561#EWQETT**";
\ No newline at end of file
+module.exports.SEED = "JSI5A6P8D9J2SAPIDJSAIPDJSAIPDJASPI48413561#EWQETT**";
```

Comment:
Horrible, even if this is not an API key. 
This should not be here please insert it into the process env